exercises/queen-attack/canonical-data.json

```
@@ -1,6 +1,6 @@
 {
   "exercise": "queen-attack",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

RE: [Test Data Versioning](https://github.com/exercism/problem-specifications#test-data-versioning) guidelines:

> MINOR version changes
> The MINOR version should change when you add functionality in a backwards-compatible manner, make non-breaking changes that alter the meaning of the test suite, make previously passing solutions possibly fail, or failing solutions fail at a different spot.
>
> There are examples of changes requiring a MINOR version change:
> - Adding or deleting test cases.
> - **Changing the test cases inputs and/or outputs.**
> - Changing the test cases ordering.
>
> MINOR changes would never break well-designed test generators, because the test-generation logic remains exactly the same.

```suggestion
  "version": "2.3.0",
```